src/.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey @ksobon. This config file is still showing as part of this PR. Can you untrack it please?